TPCCAGPUTracking/TRDTracking/AliHLTTRDTrackerComponent.cxx

```diff
@@ -201,14 +204,18 @@ int AliHLTTRDTrackerComponent::DoInit( int argc, const char** argv ) {
 
   iResult = ReadConfigurationString( arguments.Data() );
 
+  fGeo = new AliHLTTRDGeometry();
```

Comment:
You could put the other checks here, if the AliGeometry for TRD is actually available, which were with the Geometry initialization before.